### PR TITLE
Fix for extend.d.ts

### PR DESCRIPTION
**Pull request by java-4ever**
Added empty namespace, [link](https://github.com/Microsoft/TypeScript/issues/5073) to relative discussion.

### CODE REVIEW

File: extend/extend.d.ts [hunk 1]
```diff
@@ -5,5 +5,6 @@
 declare module "extend" {
 
     function extend(deepOrObject:boolean | Object, ...objectN: Object[]): any;
+    namespace extend {};
```
Comment (Brooooooklyn) [inline extend/extend.d.ts]: you should not use semicolon after namespace

**Comment by java-4ever** [comment]
@Brooooooklyn , thanks a lot, I've made the fix.

**Comment by vvakame** [comment]
sorry for delay, happy new year!

**Comment by java-4ever** [comment]
Thanks, happy New Year too :)

**Pull request merged**